src/components/MainPage/MainPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './MainPage.scss';
+
+import { user } from '../../redux/gateway';
```

Comment:
a good practice is to have `css` imports at the bottom of the import section, so will be fine move it up to line 4, and import with scss file here to line 6 ;)

Reply:
Fixed. All scss imports are now at the end of import sections